src/lib.rs

```diff
@@ -154,18 +162,24 @@ pub fn run(
     statsd: StatsdClient,
     workers: usize,
 ) -> Result<Server, std::io::Error> {
+    // Curious why you needed to use a Once synchronization for initialization
```

Comment:
The [`magick-rust`](https://github.com/nlfiedler/magick-rust#example-usage) I am using requires this initialization to give ImageMagick some global state